code/datums/mutations.dm

```diff
@@ -531,6 +531,10 @@
 /datum/mutation/human/swedish/say_mod(message)
 	if(message)
 		message = replacetext(message,"w","v")
+		message = replacetext(message,"j","y")
+		message = replacetext(message,"a",pick("�","�","�","a"))
+		message = replacetext(message,"bo","bjo")
+		message = replacetext(message,"o",pick("�","�","o"))
```

Comment:
byond uses latin1 according to http://www.byond.com/forum/?post=1077132 should work fine I think. No issues about mangled text since Paradise merged it